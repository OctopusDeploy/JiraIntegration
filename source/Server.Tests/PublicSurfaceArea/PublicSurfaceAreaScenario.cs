using System.Linq;
using Assent;
using NUnit.Framework;

namespace Octopus.Server.Extensibility.JiraIntegration.Tests.PublicSurfaceArea
{
    [TestFixture]
    public class PublicSurfaceAreaScenario
    {
        [Test]
        public void ExtensionsShouldKeepThingsPrivate()
        {
            var types = typeof(JiraIssueTrackerExtension).Assembly.GetExportedTypes().Select(t => t.FullName);
            
            this.Assent(string.Join('\n', types));
        }
    }
}