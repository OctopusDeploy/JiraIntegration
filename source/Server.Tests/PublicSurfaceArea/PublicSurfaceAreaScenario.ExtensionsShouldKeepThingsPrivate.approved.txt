Octopus.Server.Extensibility.JiraIntegration.JiraIntegrationExtension